src/main/java/nextstep/courses/domain/Size.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.courses.domain;
+
+import javax.swing.*;
+
+public class Size {
+
+    private static final int DEFAULT_WIDTH = 300;
+    private static final int DEFAULT_HEIGHT = 200;
+
+    private final int width;
+    private final int height;
+
+    public Size(int width, int height) {
+        this.width = width;
+        this.height = height;
+        validate();
```

Comment:
이미지 크기와 관련된 검증 로직이 먼저 수행되면 좋을 것 같습니다.
```suggestion
        validate();
        this.width = width;
        this.height = height;
```